jdbc/mysql-j-8/pom.xml

```diff
@@ -164,6 +164,16 @@
         </dependency>
       </dependencies>
     </profile>
+    <profile>
+      <id>native</id>
```

Comment:
Should these deps be marked as "provided" so the user can set them?

Reply:
Marking them as `provided` was what was causing one of the errors, so here we're overriding the `provided` scope with a `compile` scope.  This was the error:
```
Failures (2):
  JUnit Vintage:JdbcPostgresIamAuthIntegrationTests:pooledConnectionTest
    MethodSource [className = 'com.google.cloud.sql.postgres.JdbcPostgresIamAuthIntegrationTests', methodName = 'pooledConnectionTest', methodParameterTypes = '']
    => org.opentest4j.MultipleFailuresError: Multiple Failures (2 failures)
	java.lang.RuntimeException: Failed to get driver instance for jdbcUrl=jdbc:postgresql:///proxy_testing
	java.lang.NullPointerException: <no message>
       org.junit.vintage.engine.execution.TestRun.getStoredResultOrSuccessful(TestRun.java:200)
       org.junit.vintage.engine.execution.RunListenerAdapter.fireExecutionFinished(RunListenerAdapter.java:248)
       org.junit.vintage.engine.execution.RunListenerAdapter.testFinished(RunListenerAdapter.java:214)
       org.junit.vintage.engine.execution.RunListenerAdapter.testFinished(RunListenerAdapter.java:88)
       org.junit.runner.notification.SynchronizedRunListener.testFinished(SynchronizedRunListener.java:87)
       [...]
       Suppressed: java.lang.RuntimeException: Failed to get driver instance for jdbcUrl=jdbc:postgresql:///proxy_testing
         com.zaxxer.hikari.util.DriverDataSource.<init>(DriverDataSource.java:114)
         com.zaxxer.hikari.pool.PoolBase.initializeDataSource(PoolBase.java:331)
         com.zaxxer.hikari.pool.PoolBase.<init>(PoolBase.java:114)
         com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:108)
         com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81)
         [...]
       Caused by: java.sql.SQLException: No suitable driver
         java.sql.DriverManager.getDriver(DriverManager.java:298)
         com.zaxxer.hikari.util.DriverDataSource.<init>(DriverDataSource.java:106)
         [...]
       Suppressed: java.lang.NullPointerException
         com.google.cloud.sql.postgres.JdbcPostgresIamAuthIntegrationTests.dropTableIfPresent(JdbcPostgresIamAuthIntegrationTests.java:111)
         java.lang.reflect.Method.invoke(Method.java:566)
         org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
         org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
         org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
         [...]
  JUnit Vintage:JdbcPostgresIntegrationTests:pooledConnectionTest
    MethodSource [className = 'com.google.cloud.sql.postgres.JdbcPostgresIntegrationTests', methodName = 'pooledConnectionTest', methodParameterTypes = '']
    => org.opentest4j.MultipleFailuresError: Multiple Failures (2 failures)
	java.lang.RuntimeException: Failed to get driver instance for jdbcUrl=jdbc:postgresql:///proxy_testing
	java.lang.NullPointerException: <no message>
       org.junit.vintage.engine.execution.TestRun.getStoredResultOrSuccessful(TestRun.java:200)
       org.junit.vintage.engine.execution.RunListenerAdapter.fireExecutionFinished(RunListenerAdapter.java:248)
       org.junit.vintage.engine.execution.RunListenerAdapter.testFinished(RunListenerAdapter.java:214)
       org.junit.vintage.engine.execution.RunListenerAdapter.testFinished(RunListenerAdapter.java:88)
       org.junit.runner.notification.SynchronizedRunListener.testFinished(SynchronizedRunListener.java:87)
       [...]
       Suppressed: java.lang.RuntimeException: Failed to get driver instance for jdbcUrl=jdbc:postgresql:///proxy_testing
         com.zaxxer.hikari.util.DriverDataSource.<init>(DriverDataSource.java:114)
         com.zaxxer.hikari.pool.PoolBase.initializeDataSource(PoolBase.java:331)
         com.zaxxer.hikari.pool.PoolBase.<init>(PoolBase.java:114)
         com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:108)
         com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81)
         [...]
       Caused by: java.sql.SQLException: No suitable driver
         java.sql.DriverManager.getDriver(DriverManager.java:298)
         com.zaxxer.hikari.util.DriverDataSource.<init>(DriverDataSource.java:106)
         [...]
       Suppressed: java.lang.NullPointerException
         com.google.cloud.sql.postgres.JdbcPostgresIntegrationTests.dropTableIfPresent(JdbcPostgresIntegrationTests.java:104)
         java.lang.reflect.Method.invoke(Method.java:566)
         org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
         org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
         org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
         [...]
```